task5_TestNG/test/MyCheckTest.java

```diff
@@ -1,111 +1,109 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.NodeList;
 
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
 import java.math.BigDecimal;
 
 import static org.testng.AssertJUnit.assertEquals;
 
 public class MyCheckTest {
 
-  MyCheck myCheck;
+  private static final String PATH = "D:\\JetBrains\\IdeaProjects\\project2\\dataForTriangle.xml";
```

Comment:
I'm not happy about such hard-coded path - it's always better to resolve path in runtime